src/Themes/Sample.Theme/Recipes/DotNest.Core.SDK.Setup.recipe.json

```diff
@@ -140,10 +140,6 @@
     {
       "name": "recipes",
       "Values": [
-        {
-          "executionid": "DotNest.Core.SDK.Web",
-          "name": "Lombiq.BaseTheme.LayersAndZones"
-        },
```

Comment:
Shouldn't this be caught be the UI tests?

Reply:
What UI test?

Reply:
The SDK doesn't contain UI tests.

Reply:
I've seen this in the **Sites** SDK. It's the same problem, of course, but I think that should have a UI test at least to run the setups or something, because something being fundamentally broken like this can easily be caught like that.

Reply:
Yes I know. Sites SDK also doesn't have UI tests, which is OK I think, because DotNest covers our Site themes.
This only happens for this specific Sample theme. Do you want a simple UI test in DotNest for this Sample theme?

Reply:
Yeah, let's do that, please. Just a setup and homepage verification. This can BTW be part of this solution, actually, so people can also easily have such basic UI testing for their apps as well (I'd use it for https://github.com/Piedone/DotNest-Sites/).

Reply:
Okay, I'll add it here.

Reply:
Great, thanks.